DataConnectors/FORCEPOINT_NGFW.json

```diff
@@ -0,0 +1,116 @@
+{
+  "id": "FORCEPOINT_NGFW",
+  "title": "Forcepoint NGFW",
+  "publisher": "Forcepoint",
+  "descriptionMarkdown": "The Forcepoint NGFW (Next Generation Firewall) connector allows you to automatically export user-defined Forcepoint NGFW logs into Azure Sentinel in real-time. This enriches visibility into user activities recorded by NGFW, enables further correlation with data from Azure workloads and other feeds, and improves monitoring capability with Workbooks inside Azure Sentinel.​",
```

Comment:
When I load the json I still see weird chars after the sentence ends - Just pasting it here. Please delete some text after the "Sentinel." and retype - reload and check to see if issue disappears. 
"monitoring capability with Workbooks inside Azure Sentinel.â"